x-pack/solutions/observability/plugins/infra/public/pages/metrics/hosts/index.tsx

```diff
@@ -37,7 +37,7 @@ export const HostsPage = () => {
   ]);
 
   return (
-    <EuiErrorBoundary>
+    <KibanaErrorBoundary>
```

Comment:
Instead of individually changing some components, could we add the `<KibanaErrorBoundary />` to the [metrics_app](https://github.com/elastic/kibana/blob/main/x-pack/solutions/observability/plugins/infra/public/apps/metrics_app.tsx), wrapping the routes?

Reply:
Fixed :)